src/main/java/com/amazonaws/gurureviewercli/model/bitbucket/CodeInsightsReportData.java

```diff
@@ -0,0 +1,24 @@
+package com.amazonaws.gurureviewercli.model.bitbucket;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+/**
+ * Bitbucket CodeInsight report data.
+ */
+@Log4j2
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CodeInsightsReportData {
+
+    private String title;
+
+    private String type;
```

Comment:
maybe add a link to the Bitbucket docs where the valid types are defined?

Reply:
I'll add a link in the class JavaDoc. Per field doc might bit-rot